examples/pytorch_lightning_simple.py

```diff
@@ -0,0 +1,190 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using PyTorch Lightning.
+
+In this example, we optimize the validation accuracy of hand-written digit recognition using
+PyTorch Lightning, and MNIST. We optimize the neural network architecture as well as the optimizer
+configuration. As it is too time consuming to use the whole MNIST dataset, we here use a small
+subset of it.
+
+We have the following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python pytorch_lightning_simple.py
```

Comment:
Please add this file (`examples/pytorch_lightning_simple.py`) to the list of examples [`examples/README.md`](https://github.com/pfnet/optuna/blob/master/examples/README.md).

Reply:
Thanks, I added it to the list.